src/components/NcPasswordField/NcPasswordField.vue

```diff
@@ -96,7 +96,7 @@ export default {
 </docs>
 
 <template>
-	<NcInputField v-bind="{...$attrs, ...$props }"
+	<NcInputField v-bind="{ ...$props }"
```

Comment:
**suggestion:** pass without cloning

Is there any purpose of creating an object here? It creates an object on each re-render...

```suggestion
	<NcInputField v-bind="$props"
```

Reply:
No, this is a leftover from my debugging. It works fine without cloning.